react-frontend/codegen.ts

```diff
@@ -0,0 +1,15 @@
+import { CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+    schema: '../gateway/supergraph.graphql',
+    documents: ['src/**/*.tsx'],
+    ignoreNoDocuments: true,
+    generates: {
+        './src/gql/': {
```

Comment:
should this be named something that indicates it's' generated?
separately, a .gitattributes file indicating this path is generated would be helpful, see https://github.com/tmc/hackathon-starter/blob/master/go-graphql-backend/.gitattributes for inspo.

Reply:
gql-generated?